### PR TITLE
Fb vm

### PR DESCRIPTION
**Pull request by daniel-wtd**
- Update: Meta Files
- Fix: Deprecated commands
- Fix: Some typos
- Fix: Use new tests

- Resolves: #7, #8

### CODE REVIEW

**Comment by daniel-wtd** [comment]
@donvipre may you have a look?
@Cinux90 maybe interesting for you, too

File: kickstart/kickstart-minimal.ks [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 # This is an example for a more quite minimal kickstart configuration.
-# 
+#
 # Usage:
 # --------------------------------------------------------------------------------------
 # For some people this example can be usable at it.
 # In most cases you want to configure/review at least the following sections:
 # - User (admin) password
-#   You have to edit this line, so valid has is used. 
+#   You have to edit this line. a valid hash should be used.
```
Comment (donvipre) [inline kickstart/kickstart-minimal.ks]:
```suggestion
#   You have to edit this line. A valid hash should be used.
```

File: kickstart/kickstart-minimal.ks [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 #   You can have a look at https://github.com/while-true-do/kickstart/README.md
 # - Disk Partitions and device names
-#   This example assumes, that /dev/vd* is present and configures.
+#   This example assumes, that /dev/sd* is present and configures.
```
Comment (donvipre) [inline kickstart/kickstart-minimal.ks]:
```suggestion
#   This example assumes, that /dev/sd* is present and configured.
```

File: kickstart/kickstart-minimal.ks [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 #   If you have another device naming, you should tune these parameters.
 #   Also the sizing of the volumes should fit your needs
```
Comment (donvipre) [inline kickstart/kickstart-minimal.ks]:
```suggestion
#   The sizing of the volumes should fit your needs.
```

File: kickstart/kickstart-minimal.ks [hunk 4]
```diff
@@ -19,8 +19,6 @@
 #   DHCP is used in this example with eth0 interface. You should consider to tune this.
```
Comment (donvipre) [inline kickstart/kickstart-minimal.ks]:
```suggestion
#   DHCP is used in this example with eth0 interface. You should consider tuning this.
```

File: kickstart/kickstart-secure.ks [hunk 5]
```diff
@@ -2,21 +2,21 @@
...
 # - partition layout with secure mount options
-# 
+#
 # Usage:
 # --------------------------------------------------------------------------------------
 # For some people this example can be usable at it.
 # In most cases you want to configure/review at least the following sections:
 # - User (admin) password
-#   You have to edit this line, so valid has is used. 
+#   You have to edit this line, a valid hash should be used.
```
Comment (donvipre) [inline kickstart/kickstart-secure.ks]:
```suggestion
#   You have to edit this line. A valid hash should be used.
```

File: kickstart/kickstart-secure.ks [hunk 6]
```diff
@@ -2,21 +2,21 @@
...
 #   You can have a look at https://github.com/while-true-do/kickstart/README.md
 # - Disk Partitions and device names
-#   This example assumes, that /dev/vd* is present and configures.
+#   This example assumes, that /dev/sd* is present and configures.
```
Comment (donvipre) [inline kickstart/kickstart-secure.ks]:
```suggestion
#   This example assumes, that /dev/sd* is present and configured.
```

File: kickstart/kickstart-secure.ks [hunk 7]
```diff
@@ -2,21 +2,21 @@
...
 #   If you have another device naming, you should tune these parameters.
 #   Also the sizing of the volumes should fit your needs
```
Comment (donvipre) [inline kickstart/kickstart-secure.ks]:
```suggestion
#   The sizing of the volumes should fit your needs.
```

File: kickstart/kickstart-vm.ks [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+# --------------------------------------------------------------------------------------
+# For some people this example can be usable at it.
+# In most cases you want to configure/review at least the following sections:
+# - User (admin) password
+#   You have to edit this line.
+#   You can have a look at https://github.com/while-true-do/kickstart/README.md
+# - Disk Partitions and device names
+#   This example assumes, that only one device is present.
+#   If you have another device naming, you should tune these parameters.
+#   Also the sizing of the volumes should fit your needs
```
Comment (donvipre) [inline kickstart/kickstart-vm.ks]:
```suggestion
#   The sizing of the volumes should fit your needs.
```

File: kickstart/kickstart-vm.ks [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+# Please take care, that a plaintext password is used and this file should never
+# be used in productive environments. You can have a look in the other kickstart
+# examples for encrypting passwords.
+#
+# Usage:
+# --------------------------------------------------------------------------------------
+# For some people this example can be usable at it.
+# In most cases you want to configure/review at least the following sections:
+# - User (admin) password
+#   You have to edit this line.
```
Comment (donvipre) [inline kickstart/kickstart-vm.ks]:
```suggestion
#   You have to edit this line. A valid hash should be used.
```

**Pull request merged**